setup.py

```diff
@@ -166,6 +167,22 @@ def cmake_install_manifest_filter(manifest_files):
     ]
 
 
+def sign_development_for_apple_m1():
+    """
+    Apple enforces codesigning for arm64 targets even for local development
+    builds. See discussion here:
+        https://github.com/supercollider/supercollider/issues/5603
+    """
+    if sys.platform == "darwin" and platform.machine() == "arm64":
+        try:
+            for path in glob.glob("python/taichi/_lib/core/*.so"):
```

Comment:
hmmm this won't work if you ship the .so right? It only signs the copy in your local directory but not the one "installed" or "saved in .whl"?

Reply:
The thing is that we have to sign with *our official certificate in the future* if we want to distribute this `.so` because any local certificate is for local use only. If it's been downloaded from the internet then Apple would sigkill dyld when the user attempts to load it. I have a personal developer account so I can sign with my certificate to workaround this at least for the 1.4 release. But we ultimately have to ship it with Taichi's official account in the future, which we don't have for now. This PR is setup just for any local build to work.

Reply:
I see, yea makes sense, would you mind opening an issue to track (like we need an official account to sign binaries) ?Thanks!